### PR TITLE
Windows UNC Path Fix & GUI POC

### PR DESCRIPTION
**Pull request by apersomany**
Cleaned up some files, fixed UNC paths for windows

### CODE REVIEW

**Comment by Redblueflame** [comment]
Please fix the workspace issue.
On a clean clone, this won't work as a cargo.toml is needed.

**Comment by Redblueflame** [comment]
On another note, do not hesitate to create another PR adding the frontend and the daemon :)

**Comment by apersomany** [comment]
> 
> 
> Please fix the workspace issue.
> On a clean clone, this won't work as a cargo.toml is needed.

I'm pretty sure I only removed cargo.lock

File: .gitignore [hunk 1]
```diff
@@ -1 +1,4 @@
-/target
+**/target
+**/Cargo.lock
+.idea
+theseus_be
```
Comment (Redblueflame) [inline .gitignore:4]: This line :)

**Comment by Irgendwer01** [comment]
@apersomany why mixing both C++ and Rust?

**Comment by leocth** [comment]
> @apersomany why mixing both C++ and Rust?

flutter be like that /shrug

**Comment by Geometrically** [comment]
We've decided to use a different architecture for the UI

**Pull request closed without merging**